agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -46,23 +44,25 @@
 import com.microsoft.applicationinsights.agent.internal.wasbootstrap.configuration.Configuration.ProcessorConfig;
 import com.microsoft.applicationinsights.agent.internal.wasbootstrap.configuration.Configuration.ProfilerConfiguration;
 import com.microsoft.applicationinsights.agent.internal.wasbootstrap.configuration.RpConfiguration;
-import com.microsoft.applicationinsights.common.CommonUtils;
-import com.microsoft.applicationinsights.common.Strings;
-import com.microsoft.applicationinsights.exceptions.FriendlyException;
-import com.microsoft.applicationinsights.internal.authentication.AadAuthentication;
-import com.microsoft.applicationinsights.internal.channel.common.LazyHttpClient;
-import com.microsoft.applicationinsights.internal.config.AddTypeXmlElement;
-import com.microsoft.applicationinsights.internal.config.ApplicationInsightsXmlConfiguration;
-import com.microsoft.applicationinsights.internal.config.JmxXmlElement;
-import com.microsoft.applicationinsights.internal.config.ParamXmlElement;
-import com.microsoft.applicationinsights.internal.config.TelemetryModulesXmlElement;
-import com.microsoft.applicationinsights.internal.config.connection.ConnectionString;
-import com.microsoft.applicationinsights.internal.config.connection.InvalidConnectionStringException;
-import com.microsoft.applicationinsights.internal.profiler.GcEventMonitor;
-import com.microsoft.applicationinsights.internal.profiler.ProfilerServiceInitializer;
-import com.microsoft.applicationinsights.internal.statsbeat.StatsbeatModule;
-import com.microsoft.applicationinsights.internal.system.SystemInformation;
-import com.microsoft.applicationinsights.internal.util.PropertyHelper;
+import com.microsoft.applicationinsights.agent.internal.wascore.MetricFilter;
```

Comment:
what does `wascore `imply? or simply `core`?

Reply:
`wascore` like `wasbootstrap` 😭 it's horrible...